data/contributors.js

```diff
@@ -1444,6 +1444,15 @@ const contributors = [
     github: "https://github.com/hafeez25",
     gender: "Male",
   },
+  {
+        name : "Omosaye Moses Iseoluwa",
+        branch : "Computer Science",
+        college : "OAU",
+        year : 2023

```

Comment:
There is a , missing after 2023. Please fix that

Reply:
alright would fix that now.